### PR TITLE
Static Db class

### PR DESCRIPTION
**Pull request by joseph-jones**
We can start adding our db queries to this Static class instead of our other classes directly.

### CODE REVIEW

File: lib/cherry_silt/room.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def to_s
   def to_h
     { name: @name, description: @description, exits: @exits,
      conents: @contents, type: @type,
-     _id: @id @id.nil? ? nil : @id}.delete_if { |k,v| v.nil? }
```
Comment (joseph-jones) [inline lib/cherry_silt/room.rb:81]: did you commit this by mistake? because it doesn't work.

File: lib/cherry_silt/mongo.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+##
+module CherrySilt
+  ##
+  module Util
+    ##
+    class DB
+      @@client = nil
+
+      def self.find(query, collection)
+        @@client ||= Mongo::Client.new(['127.0.0.1:27017'], database: 'mud')
```
Comment (joseph-jones) [inline lib/cherry_silt/mongo.rb:28]: I'm thinking pulling line 28 out into it's own static method and just calling it before eventmachine startup. not sure which is a better way to do it. 

This way we'll have to have this line in every query method we have. which I don't like

File: lib/cherry_silt/room.rb (hunk 1 above)
Comment (sehardy) [inline lib/cherry_silt/room.rb:81]: Yeah, that wasn't supposed to have gone in.

**Comment by joseph-jones** [comment]
I think I found a better way to handle this. I'm working on a Mixin instead of a static class.

**Comment by sehardy** [comment]
Ooo mixins, I like it.

From: joseph-jones
Reply-To: sehardy/cherry_silt
Date: Friday, December 11, 2015 at 5:35 PM
To: sehardy/cherry_silt
Cc: Sean Hardy
Subject: Re: [cherry_silt] Static Db class (#8)

I think I found a better way to handle this. I'm working on a Mixin instead of a static class.

**Pull request closed without merging**